web/src/portico/tabbed-instructions.ts

```diff
@@ -1,8 +1,11 @@
 import $ from "jquery";
 
+import * as blueslip from "../blueslip";
 import * as common from "../common";
 
-export function detect_user_os() {
+type UserOS = "android" | "ios" | "mac" | "windows" | "linux";
```

Comment:
It would be useful to export this type for future use.

Reply:
Done!